### PR TITLE
TES Use cases used in HTGR-TES technoeconomic analysis are added.

### PR DESCRIPTION
**Pull request by JunyungKim**
--------
Pull Request Description
--------
##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)
#25 

##### What are the significant changes in functionality due to this change request?
New use cases are added. 
Point of Contact - So-bin Cho (sobin.cho@inl.gov)

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [ ] 1. Review all computer code.
- [ ] 2. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [ ] 3. Automated Tests should pass, including run_tests, pylint, and manual building tests.
- [ ] 4. If significant functionality is added, there must be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test.
- [ ] 5. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [ ] 6. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.

### CODE REVIEW

File: use_cases/2024_07_HTGR-TES_Technoeconomic Case Study/ARMA/README.rtf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{\rtf1\ansi\ansicpg1252\cocoartf2709
+\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fswiss\fcharset0 Helvetica;}
+{\colortbl;\red255\green255\blue255;}
+{\*\expandedcolortbl;;}
+\margl1440\margr1440\vieww11520\viewh8400\viewkind0
+\pard\tx720\tx1440\tx2160\tx2880\tx3600\tx4320\tx5040\tx5760\tx6480\tx7200\tx7920\tx8640\pardirnatural\partightenfactor0
+
+\f0\fs24 \cf0 This folder contains the trained ARMA files used in the M2 simulation. No changes have been made to the ARMA input files; they are essentially identical to the files available in the shared folder.}
```
Comment (PaulTalbot-INL) [inline use_cases/2024_07_HTGR-TES_Technoeconomic Case Study/ARMA/README.rtf:8]: Could you also explain the different data data sets and abbreviations you use in here, as well as what variables/years were trained with each ARMA set?

**Review by PaulTalbot-INL — CHANGES_REQUESTED** [review]
Can we add a top-level readme for the case, to explain the following:
- Who ran the case?
- Who sponsored the case?
- Who to contact about the analysis?
- What was the purpose and methodology (brief description) for the analysis?
- How is the data/runs structured?

Also a small request to define acronyms and explain the different models in the README in the ARMA folder. Otherwise, this looks great.

File: use_cases/2024_07_HTGR-TES_Technoeconomic Case Study/README.txt
Comment (PaulTalbot-INL) [inline use_cases/2024_07_HTGR-TES_Technoeconomic Case Study/README.txt:1]: This file looks like it somehow ended up being all one line. Was that your intent? Some newlines and/or whitespace might be helpful.

Comment (JunyungKim) [reply use_cases/2024_07_HTGR-TES_Technoeconomic Case Study/README.txt:1]: A new readme file is added to reflect previous comments. @PaulTalbot-INL Please let me know if you have any other suggestions or comments! :)

**Review by PaulTalbot-INL — APPROVED** [review]

**Pull request merged**